docs/multicluster/quick-start.md

```diff
@@ -74,62 +86,177 @@ antrea-multicluster   deployment.apps/antrea-mc-controller   1/1     1
 kube-system           deployment.apps/antrea-mc-controller   1/1     1            1           48s
 ```
 
-### Step 2 - initialize ClusterSet
+#### Step 2 - initialize ClusterSet
+
+Run the following commands to create a ClusterSet with cluster A to be the
+leader, and also join the ClusterSet as a member.
+
+```bash
+antctl mc init --clusterset test-clusterset --clusterid test-cluster-leader --create-token -o join-config.yml
+antctl mc join --clusterid test-cluster-leader -n kube-system --config-file join-config.yml
+```
+
+The above `antctl mc init` command creates a default token (with the
+`--create-token` flag) for member clusters to join the ClusterSet and
+authenticate to the leader cluster API server, and the command saves the token
+Secret manifest and other ClusterSet join arguments to file `join-config.yml`
+(specified with the `-o` option), which can be provided to the `antctl mc join`
+command (with the `--config-file` option) to join the ClusterSet with these
+arguments. If you want to use a separate token for each member cluster for
+security considerations, you can run the following commands to create a token
+and use the token to join the ClusterSet:
+
+```bash
+antctl mc create membertoken test-cluster-leader-token -n antrea-multicluster -o test-cluster-leader-token.yml
+antctl mc join --clusterid test-cluster-leader -n kube-system --config-file join-config.yml --token-secret-file test-cluster-leader-token.yml
```

Comment:
I tried this command, it returns below error:
```
./bin/antctl-darwin mc join --clusterid test-cluster-leader -n kube-system --config-file join-config.yml --token-secret-file test-cluster-leader-token.yml
Failed to create the Secret from the config file: Secret "" is invalid: metadata.name: Required value: name or generateName is required
Error: Secret "" is invalid: metadata.name: Required value: name or generateName is required
```
@hjiajing Could you help to check if there is code issue? or  please provide the right command.

Reply:
I tried the same command in my test env then it works well. 
```bash
❯ antctl mc join --config-file=./config.yaml --clusterid=test-cluster-east --token-secret-file ./secret.yaml  --kubeconfig /root/.kube/east
Created the Secret from the config file
ClusterClaim "id.k8s.io" created in Namespace kube-system
ClusterClaim "clusterset.k8s.io" created in Namespace kube-system
ClusterSet "test-clusterset" created in Namespace kube-system
Waiting for member cluster ready
Waiting for ClusterSet ready
```

Maybe it's because the `test-cluster-leader-token.yml` is not a valid Secret yaml file?

Reply:
The question is I am using the file generated by above step `antctl mc create membertoken test-cluster-leader-token -n antrea-multicluster -o test-cluster-leader-token.yml`.

Reply:
Removed the "config file check" when creating Secret from YAML file.  Updated the code now.